assertdb_test.go

```
@@ -0,0 +1,64 @@
+package assertdb
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	_ "github.com/lib/pq"
+	"github.com/stretchr/testify/assert"
+	"github.com/testcontainers/testcontainers-go"
+	"testing"
+	"time"
+)
+
+func TestIntegration(t *testing.T) {
+	t.Run("postgres", func(t *testing.T) {
+		ctx := context.Background()
+		req := testcontainers.ContainerRequest{
+			Image:        "postgres",
+			ExposedPorts: []string{"5432"},
+			Env:          map[string]string{"POSTGRES_PASSWORD": "passwd"},
+		}
+		postgres, _ := testcontainers.GenericContainer(ctx, testcontainers.GenericContainerRequest{
```

Avoid ignoring errors.